config/develop/nextflow-ecs-capacity.yaml

```diff
@@ -0,0 +1,11 @@
+template_path: nextflow-ecs-capacity.yaml
+stack_name: nextflow-ecs-capacity
+parameters:
+  EcsClusterName: !stack_output_external nextflow-ecs-cluster::EcsClusterName
+  AutoScalingGroupArn: !rcmd >-
+    aws autoscaling describe-auto-scaling-groups
+    | jq -r '.AutoScalingGroups | .[] | select(.AutoScalingGroupName | startswith("nextflow-ecs-cluster-EcsAutoScalingGroup")) | .AutoScalingGroupARN'
```

Comment:
This unusual way of deriving this parameter arises from the lack of ARN exposure for AutoScalingGroup -- it is not the Ref nor accessible through GetAtt.

Reply:
Should this be a comment in the config for future readers like Paolo?

Reply:
Good suggestion -- added.